pilot/model/service.go

```diff
@@ -27,6 +27,8 @@ import (
 	"fmt"
 	"sort"
 	"strings"
+
+	proxyconfig "istio.io/api/proxy/v1/config"
```

Comment:
@kyessenov do you think it makes sense to include proxy config here? We have kept this file independent so far.

Reply:
Model can include proxy config.

Reply:
So is this good to go in?